postgres/init/90-public-efhistory.sql

```diff
@@ -58,6 +58,7 @@ COPY public.__efmigrations_history_portal (migration_id, product_version) FROM s
 20220914063503_CPLP-1378-AddNewDocumentTypes	6.0.4
 20220914081150_CPLP-1406-AddAgreementAssignedApps	6.0.4
 20220916132329_CPLP-1406-AddConsentAssignedOfferSubscription	6.0.4
+20220919103742_CPLP-1408-ServiceProviderCompanyDetail	6.0.4
```

Comment:
https://github.com/catenax-ng/product-portal-backend/pull/214 does contain a migration that is likely to be merged first. This PR therefore needs to be postponed until the competing migration has been merged and applied, then this migration has to be recreated and reapplied.

Reply:
fixed